insightface/weights/weights.md

```diff
@@ -0,0 +1 @@
+Weights go here.
```

Comment:
Can we remove this file?

Reply:
Which weights are stored here exactly? And should we add this folder to the gitignore?